apps/web/vibes/soul/primitives/navigation/index.tsx

```diff
@@ -79,13 +80,13 @@ interface Props<S extends SearchResult> {
   className?: string
   isFloating?: boolean
   accountHref: string
-  cartCount?: number
+  cartCount?: Streamable<number>
```

Comment:
I think this should be `Streamable<number | null>` since the prop is optional

Reply:
Can you expand? If it's optional that means we just don't pass anything, no?

Reply:
Sometimes you don't know until the promise resolves but then the types don't let the promise have a null in it